src/stages/stage3_events.c

```diff
@@ -718,20 +706,102 @@ TASK_WITH_INTERFACE(scuttle_lethbite, BossAttack) {
 				);
 			}
 			spawn_projectile_highlight_effect(p);
-			p->move = move_linear((3 + 2 * global.diff / (float)D_Lunatic) * cdir(carg(global.plr.pos - p->pos)));
+			p->move = move_linear((3 + 2 * global.diff / (float)D_Lunatic) * (cnormalize(global.plr.pos - p->pos)));
```

Comment:
```suggestion
			p->move = move_linear((3 + (2.0 * global.diff) / D_Lunatic) * (cnormalize(global.plr.pos - p->pos)));
```

Better yet would be to move the scalar speed term into a variable and rewrite it as a `difficulty_value()` selector. This generally applies to most things that do math on `global.diff`.

But this is not really a high priority issue. It can be postponed until an actual difficulty/gameplay balance pass.